test/unit/test-amp-img-v1.js

```diff
@@ -0,0 +1,625 @@
+/**
```

Comment:
FMI: is this mostly copypaste from v0 tests?

Reply:
Correct. Same tests, slight differences for V1 mode. The idea was: we drop V0 and all old tests and just keep the new ones once launched.